src/SerializerFactory.php

```diff
@@ -186,7 +186,7 @@ public function newStatementListSerializer() {
 	 *
 	 * @since 1.4
 	 *
-	 * @return Serializer
+	 * @return DispatchableSerializer
 	 */
 	public function newStatementSerializer() {
```

Comment:
I touched this because the corresponding deserializer is already documented as "dispatchable".

This is for a historical reason: certain API modules have been designed in a way they can accept both statement as well as claim serializations. Therefor statement and claim serializers and deserializers must be dispatchable. And they are, it's just the documentation that is misleading. I think we should fix this, even if we are not using this as a feature at the moment.